modules/mobile/spec/models/authorized_services_spec.rb

```diff
@@ -0,0 +1,394 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Mobile::V0::AuthorizedServices, aggregate_failures: true do
+  let(:user) { build(:iam_user) }
```

Comment:
One thing I intend to do is change all the users away from the iam_user model. The iam_user factories are a pain to work with because they were written in a way that doesn't allow overriding specific values, which has been a recurring pain and causes us to stub everywhere. I put extra work into this to make it work with iam but still hit one or two test cases where I had to use the normal user model. 

We should be deleting iam user soon and should start moving away from it in our specs. I think this is a good place to start moving away from it. I at least want to be consistent about which one we're using here.

Reply:
I can't wait to move away from iam. Happy to move away from it in testing if it's not necessary